reactive/etcd.py

```diff
@@ -505,6 +506,30 @@ def read_tls_cert(cert):
         return data
 
 
+@when('nrpe-external-master.available')
+@when_not('nrpe-external-master.initial-config')
+def initial_nrpe_config(nagios=None):
+    set_state('nrpe-external-master.initial-config')
+    update_nrpe_config(nagios)
+
+
+@when('etcd.installed')
+@when('nrpe-external-master.available')
```

Comment:
You also need to include logic to remove the nrpe check when the relationship to etcd is removed/destroyed.